_includes/embed/bilibili.html

```diff
@@ -0,0 +1,10 @@
+<iframe
+  class="embed-video youtube"
+  loading="lazy"
+  src="https://player.bilibili.com/player.html?bvid={{ include.id }}&danmaku=1"
+  title="Bilibili video player"
+  width="100%" height="500"
+  frameborder="0"
+  allow="allow-top-navigation allow-same-origin allow-forms allow-scripts"
```

Comment:
May I ask the source of these attributes?

Reply:
I refer to youtube template and some tutorial for embedding bilibili video.After checking official embed code,I realized `title` and `allow` may be unnecessary.